ask_vopros.admin.inc

```diff
@@ -10,6 +10,11 @@
  */
 function ask_vopros_settings($form, &$form_state) {
   // If ting is configured with an agency id, just use that.
+  /**
+   * @TODO: I don't think we should force a agency id (special in tests it might
```

Comment:
Sound's good to me (was just an idea) and nice with the possibility to override it with drush.